include/dlaf/eigensolver/band_to_tridiag/mc.h

```diff
@@ -458,7 +457,9 @@ TridiagResult<T, Device::CPU> BandToTridiag<Backend::MC, D, T>::call_L(
     // Create the first max_workers workers and then reuse them.
     auto& w_pipeline = workers[sweep % max_workers];
 
-    auto dep = pika::dataflow(executor_hp, unwrapping(init_sweep), sweep, w_pipeline(), deps[0]).share();
+    auto dep = ex::make_future(
+                   dlaf::internal::transformLift(policy_hp, init_sweep, sweep, w_pipeline(), deps[0]))
+                   .share();
```

Comment:
Yes, exactly like `dataflow`. However, this comment is a good reminder to check if it actually needs to be a shared future... From a quick glance it looks like it might not need to be one, but I may be missing something.

Reply:
This one I didn't check yet... next up on my list.

Reply:
I changed it to a plain future now. I also made `copy_diag` generic so that it takes `future` or `shared_future` (it only took `shared_future` previously, that was the reason for me still using `future::share` here).